apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1330,10 +1337,13 @@ export class RushConfiguration {
   }
 
   /**
-   * The remote url of the repository. This helps "rush change" find the right remote to compare against.
+   * Remote URL(s) of the repository. If a value is provided, \"rush change\" will
+   * use one of these to find the right remote to compare against. Specifying multiple URLs
+   * is useful if a GitHub repository is renamed or for "<projectName>.visualstudio.com" vs
+   * "dev.azure.com/<projectName>" URLs.
    */
-  public get repositoryUrl(): string | undefined {
-    return this._repositoryUrl;
+  public get repositoryUrls(): string[] | undefined {
```

Comment:
Is there a meaningful difference between `undefined` versus an empty array?  The API will be easier to use if we eliminate `undefined`.

Reply:
I can do that.